plugins/single_plugins/fast-switcher.cpp

```diff
@@ -86,6 +96,9 @@ class wayfire_fast_switcher : public wf::per_output_plugin_instance_t, public wf
             return;
         }
 
+        ev->view->erase_data<animation_data_t>();
```

Comment:
I'd put these two in a separate function to avoid code duplication.
In this case, the duplicated lines are just 2, but they logically belong together (fast-switcher does not care about the view anymore), and having a single function would probably make future adjustments easier as well.